lib/saturn-node.js

```diff
@@ -114,8 +101,8 @@ async function start ({
   childProcess.on('exit', (code, signal) => {
     const reason = signal ? `via signal ${signal}` : `with code: ${code}`
     const msg = `Saturn node exited ${reason}`
-    logStream.write(msg)
-    activityStream.write({ type: 'info', message: msg })
+    console.error(msg)
+    activities.emit('activity', { type: 'info', message: msg })
```

Comment:
I think it should be enough to emit an activity. We should not need to write the message to the console, since all activities are already logged. Thoughts?

Reply:
This is what I see when I start Core with an invalid wallet address - notice the duplicated error line.

```
~/src/filstation/core on  the-great-simplification ❯ FIL_WALLET_ADDRESS=f1asd npm start

> @filecoin-station/core@10.0.2 start
> node ./bin/station.js

[24/05/2023, 14:20:59] INFO  Zinnia started.
Saturn node exited with code: 2
[24/05/2023, 14:20:59] INFO  Saturn node exited with code: 2
Saturn node closed all stdio with code 2
{
  "totalJobsCompleted": 0,
  "totalEarnings": 0
}
```

Reply:
> I think it should be enough to emit an activity. We should not need to write the message to the console, since all activities are already logged. Thoughts?

Agreed, I just missed that one. Good catch!